.env

```diff
@@ -1 +1,7 @@
 DATABASE_URL=postgres://ascii:ascii@localhost:54320/ascii-pay
+
+# Submission Port is 587 per default.
+MAIL_SENDER=payments@ascii.coffee
+MAIL_URL=mail.example.local
+MAIL_USER=username
+MAIL_PASSWORD=password
```

Comment:
Consider new line at the end of files

Reply:
Huh, this was unintentional. Surely a glitch in the editor config.